src/main.js

```diff
@@ -22,7 +22,9 @@ module.exports = config => {
   )
   const repoGitDiffHelper = new RepoGitDiff(config, metadata)
 
-  const lines = repoGitDiffHelper.getDiff(config, metadata)
+  let filteredLines = repoGitDiffHelper.getFilteredDiff(config, metadata)
+  let includedLines = repoGitDiffHelper.getIncludedFiles()
```

Comment:
**suggestion(blocking):** `let` should be `const` here

```suggestion
  const includedLines = repoGitDiffHelper.getIncludedFiles()
```

Reply:
Duh.